report/Introduction/ProjectProposal_eng.tex

```diff
@@ -0,0 +1,58 @@
+\\\\
+A common problem with both methods are that they require the customer leave their normal environment and be where the developers are. 
+This means that the customer or domain expert will have to recall the problem domain, and not be present in it.
+Since many of the details regarding the problem domain often comes as Tacit knowledge, which can make it hard to recall the details if the they are not problem domain. 
+
+The optimal scenario would be if the customer or domain expert could fill their role as Product Owner, while he or she were in the proper domain. 
+
+This would increase the quality of feedback.
+
+It is also a well known problem that customers are terrible at saying what they want, but it is very easy for them to say what they do not want.
+\\\\
+A phenomenon which Morten from this group have worked a lot with his 5 years of experience as a designer.
+In the world of design, to accommodate this problem you often present the customer with 3 proposals for designs, and ask the customer to point out what they like and do not like.
+This gives the designer some insight into what the customer actually wants and likes. 
+\\\\
+With these two problems concerning inclusion of the customer in the development process, it would be exiting to look at which other technologies could be used, that could allow the customer to express their needs while being in their problem domain while still being able to communicate with the developers. 
+
+\subsection{Execution}
+We would like to create a mobile application which will allow the customer to express their needs in an intuitive and precise manner, which at the same time would allow the requirements to be translated to a set of "acceptance tests" which would allow the developers to verify that they have fulfilled the requirements of the customer.
+This tool will also allow the developers to propose alternative solutions, not in the form of software implementations, but in the same environment as the customer. 
+This will allow the customer to look at the different problem solutions whenever they like to, and all in the same environment that the customer uses to express their requirements. 
+\\\\
+The intention is to develop a platform that will use "Finite State Automata" to express needs, requirements and wishes to a product, in a way that is useful for people without formal knowledge of these. 
+
+Using "Finite State Automata" will also allow for algorithms to highlight errors or oversights in the specified requirements.
+The use of "Finite State Automata" will also make it possible to know every state of an application, and this means that it is possible to give guarantees that applications will not end in a unwanted state. 
+
+Furthermore this would also allow that developers with far greater knowledge about "Finite State Automata" to communicate alternative solutions in a way that is transparent and understandable to the customer. 
```

Comment:
```suggestion
Lastly this would allow the developers, with far greater knowledge about "Finite State Automata", to communicate alternative solutions in a way that is transparent and understandable to the customer. 
```